modules/Staff/applicationForm_jobOpenings_view.php

```diff
@@ -67,10 +67,10 @@
         echo "<a href='".$_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/'.$_SESSION[$guid]['module']."/applicationForm.php'>".__('Submit Application Form')."<img style='margin-left: 5px' title='".__('Submit Application Form')."' src='./themes/".$_SESSION[$guid]['gibbonThemeName']."/img/plus.png'/></a>";
         echo '</div>';
```

Comment:
Please also move these to the template.

Reply:
Thanks Koala. Interestingly, this is one of the few areas where an Action link occurs without a form or table. Ideally I'd like them to use the same object, so we'll keep this one out of the template for now and look at a more object-oriented solution.